CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.5
```

Comment:
Let's start adding a `1.0.0 (unreleased)` as a collecting TODO, in this case killing symbols.